Create 2025-02 newsletter (#44)
OBF/newsletter: newsletters/2025-02.md

### OBF Event Fellowships

Given the general cost-of-living increases and their impact on travel and event attendance, the OBF board has decided to **increase the maximum amount of funding that OBF Event Fellowships can cover to US $1,500**.    

Next deadline: April 1, 2025

The current application period for the OBF Event Fellowship lasts until April 1, 2025, so there is still time to [get your application submitted](https://forms.gle/3yLg94G1C9nkZkvq5).

These fellowships are ideal to support your virtual or in-person participation in conferences related to open source bioinformatics or open science (in-person or virtual), and potentially additional expenses such as travel, lodging, childcare for the duration of the event, small hardware (e.g. a headset or webcam), or high-speed internet.

You can read more about [applying for an OBF Event Fellowship](https://www.open-bio.org/event-awards/#fellowships-applications), and [the experiences of past awardees](https://www.open-bio.org/category/travel-fellowship/event-fellowship/).

> 

Reviewer: ```suggestion

### GSoC 2025 Call For Projects

OBF is excited to apply to GSoC again this year and welcomes project ideas from the community. This year, the projects can be of various lengths: small (~90 hours), medium (~175 hours), or large (~350 hours). The standard coding period will be 12 weeks for medium and large projects and 8 weeks for small projects.

If you have a project idea, please reach out via the mentors and admins mailing list (https://groups.google.com/g/obf-gsoc-mentors). Please refer to the project ideas from 2022 for inspiration: [https://www.open-bio.org/events/gsoc/gsoc-project-ideas/](https://www.open-bio.org/events/gsoc/gsoc-project-ideas/).

Submission deadline: February 8, 2025

```

---
tags: obf
---

_Having trouble viewing this email or looking for a permalink? [View this newsletter in your browser](https://github.com/OBF/newsletter/blob/master/newsletters/2025-02.md)._

_This email was sent to the [bosc-announce](https://groups.google.com/g/bosc-announce) and [open-bio-l](http://mailman.open-bio.org/mailman/listinfo/open-bio-l/) mailing lists._

# OBF Newsletter February 2025

TL;DR: Updates from BOSC 2025, changes to the OBF fellowship and community news!

In longer form: 

## OBF News

### BOSC news

#### BOSC 2025

> This year, BOSC will take place July 21-22, 2025 in Liverpool, UK (as part of ISMB/ECCB 2025) and will cover all aspects of open source bioinformatics and open science.

Reviewer 2: ```suggestion
Our annual conference, the [Bioinformatics Open Source Conference](https://www.open-bio.org/events/bosc-2025/) (BOSC), will take place July 21-22, 2025 in Liverpool, UK as part of ISMB/ECCB 2025. This will be the 26th BOSC; as always, it will cover all aspects of open source bioinformatics and open science.
```

The second day of BOSC will be a joint session with the newly-renamed Bio-Ontologies and Knowledge Representation. 
This day will feature a joint keynote and talks on some of our favorite topics, including open data and reusable and reproducible science.

> 

Author: Great, added that in too!

These fellowships are ideal to support your virtual or in-person attendance of conferences related to open source bioinformatics or open science (in-person or virtual), and potentially additional expenses such as travel, lodging, childcare for the duration of the event, small hardware (e.g. a headset or webcam), or high-speed internet.

You can read more about [applying for an OBF Event Fellowship](https://www.open-bio.org/event-awards/#fellowships-applications), and [the experiences of past awardees](https://www.open-bio.org/category/travel-fellowship/event-fellowship/).

## Community updates

### sandbox.bio: Command-line tutorials

[sandbox.bio](https://sandbox.bio/) is a platform for bioinformatics command-line tutorials. 
Everything runs in your browser so no setup is required. 

Here are a few of their recent tutorials:

* [Wrangling FASTA/FASTQ files with SeqKit](https://sandbox.bio/tutorials/seqkit-intro)
* [Command-line exercises from a Carpentries tutorial](https://sandbox.bio/tutorials/carpentries-shell-novice)
* [Introduction to sequence alignment with BLAST](https://sandbox.bio/tutorials/blast-intro)

> iohackathons page, [https://biohackathons.github.io](https://biohackathons.github.io), lists more upcoming bioinformatics-related hackathons and similar events. And you can also [suggest your own additions](https://github.com/biohackathons/biohackathons.github.io/issues)! 

Reviewer 2: ```suggestion
The biohackathons page, [https://biohackathons.github.io](https://biohackathons.github.io), lists more upcoming bioinformatics-related hackathons and similar events. And you can also [suggest your own additions](https://github.com/biohackathons/biohackathons.github.io/issues)! 
```

---
tags: obf
---

_Having trouble viewing this email or looking for a permalink? [View this newsletter in your browser](https://github.com/OBF/newsletter/blob/master/newsletters/2025-02.md)._

_This email was sent to the [bosc-announce](https://groups.google.com/g/bosc-announce) and [open-bio-l](http://mailman.open-bio.org/mailman/listinfo/open-bio-l/) mailing lists._

# OBF Newsletter February 2025

> TL;DR: Updates from BOSC 2025, changes to the OBF fellowship and community news!

Reviewer 3: ```suggestion
TL;DR: Updates from BOSC 2025; Call for Projects for OBF's participation in GSoC 2025; changes to the OBF fellowship; and community news!
```

Reviewer 2: s/Call/call

> changes to the OBF fellowship;
that sounds like more changes than there actually are. Maybe just make it "OBF event award news"?

In longer form: 

## OBF News

### BOSC news

#### BOSC 2025

Our annual conference, the [Bioinformatics Open Source Conference](https://www.open-bio.org/events/bosc-2025/) (BOSC), will take place July 21-22, 2025 in Liverpool, UK as part of ISMB/ECCB 2025. This will be the 26th BOSC; as always, it will cover all aspects of open source bioinformatics and open science.

> The second day of BOSC will be a joint session with the newly-renamed Bio-Ontologies and Knowledge Representation. 

Reviewer 3: ```suggestion
The second day of BOSC will be a joint session with the 2025 meeting of the newly-renamed [Bio-Ontologies and Knowledge Representation](https://www.bio-ontologies.org.uk/2025-meeting). 
```

This day will feature a joint keynote and talks on some of our favorite topics, including open data and reusable and reproducible science.

[Abstract submission is open until April 17 for talks and posters](https://www.open-bio.org/events/bosc-2025/submit/) (and then until May 15 for late posters). Some registration fee support is available for presenters; you can request this on the abstract submission form.

To keep up with BOSC news, please follow us on [LinkedIn](https://www.linkedin.com/groups/14344023/), [Bluesky](https://bsky.app/profile/bosc.bsky.social), [Mastodon](https://genomic.social/@BOSC), and [Slack](https://join.slack.com/t/obf-bosc/shared_invite/zt-n5ur1gsj-z2C~69_4lYTFPg5tbWA8Ew)! (We are no longer on Twitter/X; [find out why](https://www.open-bio.org/2023/11/20/leaving-x/).)

### OBF Event Fellowships

> Given the general cost-of-living increases and their impact on travel and event attendance, the OBF board has decided to **increase the maximum amount of funding that OBF Event Fellowships can cover to US $1,500**.    

Reviewer 3: Why not say that this is a result of inflation. The impact of CoL increases on travel expenses seems indirect, and the fellowships don't generally cover costs of living (and even if you count meal expenses, the majority of the increase is to cover increased costs for air fares, lodging etc, which aren't costs of living).

Reviewer 2: Not sure what you mean here. "CoL" is a general way to refer to increased costs. "Inflation" has unfortunately become a politicized term, so I think Bastian was wise to avoid it.

If you want to be even more guarded, you could just leave out the reason and say that we've increased the max funding amount.

Author: I'll rephrase to "Given that the costs for travel and event attendances have increased in the past years", which focuses on the immediate reason we increase the award level, without focusing on the reasoning for those increases.
